core/model/schema/modx.sqlsrv.schema.xml

```diff
@@ -703,7 +685,7 @@
         models to be loaded prior to modX request handling. Useful for when you want to integrate custom data and
         extensions, such as Custom Resource Classes, into the core.
     -->
-    <object class="modExtensionPackage" table="extension_packages" extends="xPDO\Om\xPDOObject">
+    <object class="modExtensionPackage" table="extension_packages" extends="xPDO\Om\xPDOSimpleObject">
```

Comment:
What is this change doing in here? This would need an upgrade script if you are adding an auto_increment primary key to this table.

Reply:
It fixes sqlsrv schema issue, because actually in the code it extends xPDOSimpleObject.

https://github.com/modxcms/revolution/blob/3.x/core/src/Revolution/modExtensionPackage.php#L23
https://github.com/modxcms/revolution/blob/3.x/core/src/Revolution/mysql/modExtensionPackage.php#L13

But sqlsrv implementation extends  xPDOObject. https://github.com/modxcms/revolution/blob/3.x/core/src/Revolution/sqlsrv/modExtensionPackage.php#L13

So, question where is an issue. Mysql seems configures properly, sqlsrv probably works, maybe not, but who cares? :)

Reply:
Doh! I didn't even notice the sqlsrv schema thing. And yes, who cares, indeed!